actions/kustomize-gke-deploy/action.yaml

```diff
@@ -15,8 +15,10 @@ runs:
     - name: Deploy kustomized crds
       shell: bash
       run: |
-        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl apply -f -
-        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl wait --for condition=established --timeout=${{ inputs.timeout }}s -f -
+        if [ ! -z "$(kubectl kustomize ${{ inputs.kustomization-path }}  | kfilt -i kind=CustomResourceDefinition)" ]; then
```

Comment:
Maybe extract into a variable? You use it 3 times, and since `kubectl kustomize` can become rather complex, this could save some time